guacamole/src/main/frontend/src/app/rest/services/tunnelService.js

```diff
@@ -54,6 +54,15 @@ angular.module('rest').factory('tunnelService', ['$injector',
      */
     var DOWNLOAD_CLEANUP_WAIT = 5000;
 
+    /**
+     * The maximum size a chunk may be during uploadToStream()
```

Comment:
Minor nitpick: Block comments should end in a period.

Also it would be really helpful if this specified what units this chunk size is in.